.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 12.18.3
```

Comment:
Can this be the latest node 14 instead.

Reply:
Nevermind, you can delete this as this PR already solves that https://github.com/appfolio/react-gears/pull/1027/files